server/lib/wca_live/competitions/qualification.ex

```diff
@@ -6,20 +6,20 @@ defmodule WcaLive.Competitions.Qualification do
   use WcaLive.Schema
   import Ecto.Changeset
 
-  @required_fields [:type, :when, :attempt_result]
+  @required_fields [:type, :when_date, :level]
   @optional_fields []
 
   @primary_key false
   embedded_schema do
     field :type, :string
-    field :when, :utc_datetime
-    field :attempt_result, :integer
+    field :when_date, :date
+    field :level, :integer
   end
```

Comment:
Note to self: the qualification is stored as a JSON column in the database, and currently it's empty for all competitions (since the WCA website doesn't support it yet), therefore it's safe to change the schema without any migrations.